test/rest_api/test_task_manager.py

```diff
@@ -0,0 +1,123 @@
+import sys
+
+from rest_util import new_test_module, new_test_task
+
+def test_task_manager_modules(rest_api):
+    with new_test_module(rest_api):
+        resp = rest_api.send("GET", "task_manager/list_modules")
+        resp.raise_for_status()
+        modules = resp.json()
+        assert "test" in modules, "test module was not listed"
+        assert "repair" in modules, "repair module was not listed"
```

Comment:
let's keep the test focused on the test module at this stage, other modules are of no concern for the scope of this test.